wiki/modding/README.md

```diff
@@ -4,7 +4,7 @@ Beat Saber _**does not**_ have built in mod support.
 
 Instead, most mods within the mod installer rely on [BSIPA (Beat Saber Illusion Plugin Architecture)](https://github.com/nike4613/BeatSaber-IPA-Reloaded/) to inject plugins into the game, as well as providing some useful tools for us modders.
 
-Previously, we used [IPA (Illusion Plugin Architecture)](https://github.com/Eusth/IPA). Some plugins still use this, and is temporarily supported by BSIPA. We still use IPA for the mod tutorial, however it is recommended to use BSIPA for any future modding endeavors.
+Previously, we used [IPA (Illusion Plugin Architecture)](https://github.com/Eusth/IPA). Some plugins still use this, and is temporarily supported by BSIPA.
```

Comment:
Because IPA is barely used these days (iirc the only mods that use it are Arti's custom text mods, which have not been updated), perhaps consider removing this bit entirely?